airbyte-webapp/src/views/Connection/CatalogTree/components/BulkHeader.tsx

```diff
@@ -103,7 +103,7 @@ export const BulkHeader: React.FC<BulkHeaderProps> = ({ destinationSupportedSync
       <CheckboxCell />
       <ArrowCell />
       <HeaderCell flex={0.4}>
-        <Toggle small checked={options.selected} onChange={() => onChangeOption({ selected: !options.selected })} />
+        <Switch small checked={options.selected} onChange={() => onChangeOption({ selected: !options.selected })} />
```

Comment:
This is an example of an async effect caused by a sync option. Hard to know what Switch triggered things.